pkgs/top-level/all-packages.nix

```diff
@@ -23698,21 +23698,23 @@ with pkgs;
 
   wcslib = callPackage ../development/libraries/science/astronomy/wcslib { };
 
-  webkitgtk = callPackage ../development/libraries/webkitgtk {
+  webkitgtk_4_0 = callPackage ../development/libraries/webkitgtk {
     harfbuzz = harfbuzzFull;
     inherit (gst_all_1) gst-plugins-base gst-plugins-bad;
     inherit (darwin) apple_sdk;
   };
 
-  webkitgtk_4_1 = webkitgtk.override {
+  webkitgtk_4_1 = webkitgtk_4_0.override {
     libsoup = libsoup_3;
   };
 
-  webkitgtk_6_0 = webkitgtk.override {
+  webkitgtk_6_0 = webkitgtk_4_0.override {
     libsoup = libsoup_3;
     gtk3 = gtk4;
   };
 
+  webkitgtk = webkitgtk_4_0;
```

Comment:
Unsure if this would be preferable to move to aliases.nix with a warn to explicitly set the ABI version instead of relying on whichever is provided by default. (I think this would be better)

Reply:
I feel like I've seen differing opinions on this as to whether to encourage version pinning or not, to avoid having to maintain many versions of programs.

Reply:
Another thing is that the three abi versions (4_0, 4_1, 6_0) are on the same version of webkitgtk, those suffixes don't indicate the version of webkitgtk being used.

Reply:
Merging this and opening a different PR suggesting a move to a warn.

Reply:
Amending this